CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.2] - 2024-08-07
+### Changed
+- Add MWAA 2.4.3 support
+- Backup and restore of `XCOM` table
```

Comment:
We use it to pass variables between different tasks. I believe the table is necessary in case we want to review the task history. For example, a task can perform some validations, and the next task sends a notification based on the validation results. We would use XCOM to pass the results from the validation task to the notification task. If the table is not backed up, we cannot easily review the XCOM history.

I add Copilot's response why the table should be backed up for your reference:

> Backing up the XCOM table is crucial when migrating Airflow to another system for several reasons:
> 
> **Historical Data**: The XCOM table contains historical data about task executions, which can be valuable for debugging and auditing purposes.
> **Continuity**: Ensuring that the data passed between tasks is preserved helps maintain the continuity of your workflows. Without this data, tasks that depend on previous results might fail or produce incorrect outcomes.
> **Validation and Notifications**: As you mentioned, tasks often perform validations and send notifications based on those results. Losing the XCOM data would mean losing the context needed for these operations.
> Troubleshooting: If issues arise after migration, having access to the XCOM history can help troubleshoot and resolve problems more efficiently.

Reply:
Can I resolve this comment @crupakheti please ?

Reply:
Got it, thank you! Yes, please feel free to resolve!